chapters/en/slides/module5/module5_05.Rmd

```diff
@@ -0,0 +1,393 @@
+---
+params:
+  dynamictitle: "module5_05"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+
+# Effective use of color for quantitative data
+
+Notes:
+This slide deck describes how to choose color to accurately represent numerical data.
+
+---
+
+## Repetition of hue, lightness, and saturation
+
+<center><img src="/module5/hsl-questions.png" width="50%"></img></center>
```

Comment:
This is too small I think.

Reply:
I was actually worried that this was too big, but ok making it a bit bigger